src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManager.cs

```diff
@@ -40,6 +42,11 @@ internal class ServiceLifetimeManager<THub> : ServiceLifetimeManagerBase<THub> w
             }
 #endif
             _clientConnectionManager = clientConnectionManager;
+
+            if (hubOptions.Value.SupportedProtocols != null && hubOptions.Value.SupportedProtocols.Any(x => x.Equals(Constants.Protocol.BlazorPack, StringComparison.OrdinalIgnoreCase)))
```

Comment:
can the protocol add to `globalHubOptions` only?

Reply:
It's set with `AddServerSideBlazor()`(https://github.com/dotnet/aspnetcore/blob/master/src/Components/Server/src/DependencyInjection/ComponentServiceCollectionExtensions.cs#L48) which only affected `ComponentHub`.